libnmstate/nm/sriov.py

```diff
@@ -67,13 +68,15 @@ def get_info(device):
     if not connection.profile:
```

Comment:
The docstring says this gets the live configuration but with this change it is more like the runtime state IMHO. Also given that it checks for the path in the sysfs, it is not clear why this functions checks for the presence of a connection profile since it does not matter anymore, does it?

Reply:
It does not matter anymore, I will remove it. Thank you!

Reply:
Comments below!